lib/linodeapi/raw.rb

```diff
@@ -82,10 +82,16 @@ def make_call(method, *args)
     def call(method, params = {})
       options = build_call_body(method, params)
       self.class.error_check self.class.post(
-        '', body: options, base_uri: endpoint
+        '', body: options,
+            base_uri: endpoint,
+            headers: { 'User-Agent' => user_agent }
       )
     end
 
+    def user_agent
+      "linodeapi/#{LinodeAPI::VERSION}"
```

Comment:
Should this be amended with some form of “ruby” or “gem”? I can imagine there being other “linodeapi” implementations that could totally feasibly use the same user-agent as this

Reply:
Good idea.  I added `ruby` as well as the Ruby version to avoid clashing with other languages & get visibility into the language versions that are in use as well.